microraiden/microraiden/test/fixtures/client.py

```diff
@@ -48,7 +48,6 @@ def client(
             client,
             private_keys,
             channel_manager_address,
-            balance=0
```

Comment:
@pcppcp withdraw was failing because of this: although a withdraw was made in the test, teardownn tried to set balance=0 on close, which made the tx fails. Is it ok to simply remove it, avoiding updating the balance in the client, and closing with current balance?